app/model/abstractelement.js

```diff
@@ -0,0 +1,130 @@
+// #! /usr/bin/env python
+// # -*- coding: utf-8 -*-
+// # vim:fenc=utf-8
+// #
+// # Copyright © 2019 Michael Bittencourt <mchl.bittencourt@gmail.com>
+// #
+// # Distributed under terms of the MIT license.
+
+const etree = require('xmlbuilder');
+
```

Comment:
add comments in order to explain the importance to create this class  its  a recommendation